diff --git a/build_tools/build_p4c_with_p4mlir_ext.sh b/build_tools/build_p4c_with_p4mlir_ext.sh
index 1b30d52..bb08b5a 100755
--- a/build_tools/build_p4c_with_p4mlir_ext.sh
+++ b/build_tools/build_p4c_with_p4mlir_ext.sh
@@ -10,9 +10,9 @@ set -ex
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )  # p4mlir/build_tools
 P4MLIR_REPO_DIR=$( cd "$SCRIPT_DIR"/.. &> /dev/null && pwd )
 
-LLVM_REPO_DIR=$P4MLIR_REPO_DIR/third_party/llvm-project
+LLVM_REPO_DIR=$P4MLIR_REPO_DIR/../../third_party/llvm
 LLVM_BUILD_DIR=$LLVM_REPO_DIR/build
-LLVM_INSTALL_DIR=$P4MLIR_REPO_DIR/install
+LLVM_INSTALL_DIR=$LLVM_REPO_DIR/install
 
 P4C_REPO_DIR=$P4MLIR_REPO_DIR/third_party/p4c
 P4C_BUILD_DIR=$P4C_REPO_DIR/build
